docs/getting_started.rst

```diff
@@ -1,9 +1,75 @@
 Getting Started
 ===============
+Installing sparcle_qc
+---------------------
+To get started, you will need to install sparcle_qc and its dependencies. We have provided a yaml file to create a conda environment containing all of the necessary packages.
 
+First, clone the `sparcle_qc repository <https://github.com/carolinesargent/sparcle_qc>`_
```

Comment:
would be cool if we could make this code to be copied (git clone https:/github......)